rlxnix/utils/mappings.py

```diff
@@ -1,21 +1,21 @@
 from enum import Enum
 
 import nixio
-import nixio
 
 
 class DataType(Enum):
       Continuous = 0
       Event = 1
       StimulusSegment = 2
 
-
 type_map = {1.0: {DataType.Event: "nix.events.position",
                   DataType.Continuous: "nix.data.sampled",
                   DataType.StimulusSegment: "nix.event.stimulus"},
-            1.1: {DataType.Event: "relacs.data.event",
-                  DataType.Continuous: "relacs.data.sampled",
-                  DataType.StimulusSegment: "relacs.stimulus"}
+
```

Comment:
will your changes break with 1.0 files? Now it works with the lists for the 1.1 files, does this affect the 1.0 datasets?
Better convert them to list of strings as well?